source/helpers/hotfix.ts

```diff
@@ -16,21 +18,30 @@ function parseCsv(content: string): string[][] {
 	return lines;
 }
 
-export type HotfixStorage = Array<[FeatureID, string]>;
-
-export const updateHotfixes = cache.function(async (version: string): Promise<HotfixStorage> => {
+async function fetchHotfix(path: string): Promise<string> {
 	// The explicit endpoint is necessary because it shouldn't change on GHE
 	// We can't use `https://raw.githubusercontent.com` because of permission issues https://github.com/refined-github/refined-github/pull/3530#issuecomment-691595925
-	const request = await fetch('https://api.github.com/repos/refined-github/refined-github/contents/hotfix.csv?ref=hotfix');
+	const request = await fetch(`https://api.github.com/repos/refined-github/yolo/contents/${path}`);
 	const {content} = await request.json();
 
 	// Rate-limit check
+	if (content) {
+		return atob(content).trim();
+	}
+
+	return '';
+}
+
+export type HotfixStorage = Array<[FeatureID, string]>;
+
+export const updateHotfixes = cache.function(async (version: string): Promise<HotfixStorage> => {
+	const content = await fetchHotfix('broken-features.csv');
 	if (!content) {
 		return [];
 	}
 
 	const storage: HotfixStorage = [];
-	for (const [featureID, unaffectedVersion, relatedIssue] of parseCsv(atob(content))) {
+	for (const [featureID, relatedIssue, unaffectedVersion] of parseCsv(content)) {
```

Comment:
The order has been updated:

https://github.com/refined-github/yolo/blob/main/broken-features.csv


<img width="352" alt="Screen Shot 4" src="https://user-images.githubusercontent.com/1402241/176994478-7dbdef46-b7bb-4b23-a9f6-9d020980dba8.png">

Reply:
FS